napari/_qt/qt_event_filters.py

```diff
@@ -0,0 +1,24 @@
+"""Qt event filters providing custom handling of events."""
+
+import html
+
+from qtpy.QtCore import QEvent, QObject, Qt
+from qtpy.QtWidgets import QWidget
+
+
+class QtToolTipEventFilter(QObject):
+    """
+    An event filter that converts all plain-text widget tooltips to rich-text
+    tooltips.
+    """
+
+    def eventFilter(self, qobject: QObject, event: QEvent) -> bool:
+        if event.type() == QEvent.ToolTipChange and isinstance(
+            qobject, QWidget
+        ):
+            tooltip = qobject.toolTip()
+            if tooltip and not Qt.mightBeRichText(tooltip):
```

Comment:
this might be an issue... I had to work around `mightBeRichText` at magicgui https://github.com/napari/magicgui/blob/3803ea5654e09c7d5f44867223b16955fb3bc2a6/magicgui/backends/_qtpy/widgets.py#L950-L963 ...  so should add a test here to make sure this is at least getting tested on both backends

Reply:
Will do !